libcxx/include/__flat_map/flat_map.h

```diff
@@ -52,10 +53,9 @@
 #include <__type_traits/maybe_const.h>
 #include <__utility/exception_guard.h>
 #include <__utility/pair.h>
+#include <__vector/vector.h>
 #include <initializer_list>
 #include <stdexcept>
-#include <string>
```

Comment:
I wonder how we ended up including `string`?

Reply:
Good question. I think it might be the case where originally we are throwing some specific exceptions that tells users the flat_map has been cleared after a failed operation which ends up creating some strings. But that part has been removed and the string header might be left over